### PR TITLE
Added files to excel2cha CLAN header generator

### PR DESCRIPTION
**Pull request by jttaufa**
Been changing files and folder structure around but should finally have a starting point.

### CODE REVIEW

File: scripts/excel2cha/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	```python generate_headers.py```
 	OR
 	2. Running the file in Python IDLE
-3. At the prompt, select the excel file to use. It must have a `Sessions` and a `Participants` sheet with at least the following headers:
+3. At the prompt, select the excel file to use. A sample spreadsheet can be found in the `sample/` folder. It must have a `Sessions` and a `Participants` sheet with at least the following headers:
```
Comment (benfoley) [inline scripts/excel2cha/README.md]: Hi, test comment

Comment (benfoley) [inline scripts/excel2cha/README.md]: hmm, test start a review

Comment (jttaufa) [reply scripts/excel2cha/README.md]: Hello, test response

**Review by benfoley — COMMENTED** [review]
great work

File: scripts/excel2cha/README.md (hunk 1 above)
Comment (benfoley) [reply scripts/excel2cha/README.md]: @fauxneticien test personal comment

**Comment by benfoley** [comment]
Let's add an option to pass spreadsheet filename as an arg for command line usage.

**Pull request merged**